modules/dr-recover-lost-control-plane-hosts.adoc

```diff
@@ -31,10 +33,10 @@ This procedure assumes that you have copied a snapshot file called `snapshot.db`
 
 .. Run the `etcd-snapshot-restore.sh` script.
 +
-Pass in two parameters to the `etcd-snapshot-restore.sh` script: the path to the backed up etcd snapshot file and list of members, which is defined by the `INITIAL_CLUSTER` variable.
+Pass in two parameters to the `etcd-snapshot-restore.sh` script: the path to the etcd backup file and list of members, which is defined by the `INITIAL_CLUSTER` variable.
 +
 ----
-[core@ip-10-0-143-125 ~]$ sudo /usr/local/bin/etcd-snapshot-restore.sh /home/core/snapshot.db $INITIAL_CLUSTER
+[core@ip-10-0-143-125 ~]$ sudo /usr/local/bin/etcd-snapshot-restore.sh /home/core/snapshot_db_kuberesources_<datetimestamp>.tar.gz $INITIAL_CLUSTER
```

Comment:
Hello @retroflexer, Is the command line parameter changed from: 'snapshot.db' to 'snapshot_db_kuberesources_<datetimestamp>.tar.gz' ?  and 'snapshot_db_kuberesources_<datetimestamp>.tar.gz'  is zip file of directory: './asset', right?
if yes, is this change will be put in use from 4.3 only? or backport to 4.1,4.2?
pls correct me if there is any misunderstanding, thanks in advance!
FYI @bergerhoffer

Reply:
This 4.3 and beyond. Doesn't apply to 4.1 or 4.2. For backward compatibility, we allow specifying snapshot.db as well, if they have older backups (from 4.1 or 4.2).